app/services/process_apprenticeship_bulletin.rb

```diff
@@ -0,0 +1,43 @@
+class ProcessApprenticeshipBulletin
+  def self.call(uri:, title:, date:)
+    new(uri:, title:, date:).call
+  end
+
+  def initialize(uri:, title:, date:)
+    @uri = uri
+    @title = title
+    @date = date
+  end
+
+  def call
+    standards_import = StandardsImport.where(
+      name: uri,
+      organization: title
+    ).first_or_initialize(
+      notes: "From Scraper::ApprenticeshipBulletinsJob",
+      public_document: true,
+      source_url: Scraper::ApprenticeshipBulletinsJob::BULLETIN_LIST_URL,
+      bulletin: true,
+      metadata: {date: date}
+    )
+
+    if standards_import.new_record?
+      standards_import.save!
+
+      filename = File.basename(URI.decode_uri_component(uri))
+      if standards_import.files.attach(io: URI.parse(uri).open, filename: filename)
+        import = standards_import.imports.create(
```

Comment:
Since the goal is to remove `standards_import.files`, what do you think of centering this around the `standards_import.imports` instead? Something like:


```ruby
import = standards_import.imports.create!(
  type: "Imports::Uncategorized",
  file: URI.parse(uri).open,
  public_document: true,
  metadata: standards_import.metadata
)
import.process(listing: true)

standards_import.files.attach(standards_import.imports.first.file)
```

This way we can easily delete that last line later.